### ISSUE TITLE
Adobe Comment on 2.4.11 Focus Appearance (minimum)

### ISSUE THREAD

**Issue by awkawk** [w3c/wcag#1887]
This SC is very confusing to new readers, but we are not sure how to simplify it, which puts a large burden on the understanding document. The WG should be particularly receptive to suggestions for improvements to the SC text and the Understanding content.

**Comment by alastc** [comment]
In progress in #2000

**Comment by alastc** [comment]
Proposed response:

<hr>

After <a href="https://github.com/w3c/wcag/pull/2000" target="_blank">PR 2000</a> introduced a new part of the intent aimed at providing simple examples, we think this issue could be closed. The group will remain responsive to improvements.

**Comment by alastc** [comment]
In the meeting today we agree to implement #2070, which is our attempt to simplify the SC text more.

The group remains open to other ways the SC could be simplified (whilst having the same scope).
